From 491edd72d3cbbf789a0b3b668cd177637c63acd6 Mon Sep 17 00:00:00 2001
From: Christoph Jabs <christoph.jabs@helsinki.fi>
Date: Fri, 4 Oct 2024 11:47:48 +0300
Subject: [PATCH] modify entry point

---
 setup.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index fb241d4..0800756 100755
--- a/setup.py
+++ b/setup.py
@@ -9,6 +9,7 @@ setup(name='gbd_tools',
   author='Markus Iser',
   author_email='markus.iser@kit.edu',
   packages=[
+    "gbd_main",
     "gbd_core", 
     "gbd_init",
     "gbd_server"
@@ -36,7 +37,7 @@ setup(name='gbd_tools',
   ],
   entry_points={
     "console_scripts": [
-        "gbd = gbd:main"
+        "gbd = gbd_main:main"
     ]
   }
 )
-- 
2.46.0

